resources/rest-service/cloudify/migrations/versions/a6d00b128933_4_4_to_4_5.py

```diff
@@ -18,25 +17,184 @@
+        u'users_tenants_tenant_id_fkey', 'users_tenants', type_='foreignkey')
+    op.drop_constraint(
+        u'users_tenants_role_id_fkey', 'users_tenants', type_='foreignkey')
+    op.create_foreign_key(
+        op.f('users_tenants_tenant_id_fkey'),
+        'users_tenants',
+        'tenants', ['tenant_id'], ['id'],
+        ondelete='CASCADE')
+    op.create_foreign_key(
+        op.f('users_tenants_user_id_fkey'),
+        'users_tenants',
+        'users', ['user_id'], ['id'],
+        ondelete='CASCADE')
+    op.create_foreign_key(
+        op.f('users_tenants_role_id_fkey'),
+        'users_tenants',
+        'roles', ['role_id'], ['id'],
+        ondelete='CASCADE')
+
+    # set null=true
+    op.alter_column(
+        'groups_tenants', 'role_id', existing_type=sa.INTEGER(), nullable=True)
+
+    # dep_up blueprint fks
+    op.create_foreign_key(
+        op.f('deployment_updates__old_blueprint_fk_fkey'),
+        'deployment_updates',
+        'blueprints', ['_old_blueprint_fk'], ['_storage_id'],
+        ondelete='CASCADE')
+    op.create_foreign_key(
+        op.f('deployment_updates__new_blueprint_fk_fkey'),
+        'deployment_updates',
+        'blueprints', ['_new_blueprint_fk'], ['_storage_id'],
+        ondelete='CASCADE')
+
+    # adding tenant_id indexes
+    op.create_index(
+        op.f('blueprints__tenant_id_idx'),
+        'blueprints', ['_tenant_id'],
+        unique=False)
+    op.create_index(
+        op.f('deployment_modifications__tenant_id_idx'),
+        'deployment_modifications', ['_tenant_id'],
+        unique=False)
+    op.create_index(
+        op.f('deployment_update_steps__tenant_id_idx'),
+        'deployment_update_steps', ['_tenant_id'],
+        unique=False)
+    op.create_index(
+        op.f('deployment_updates__tenant_id_idx'),
+        'deployment_updates', ['_tenant_id'],
+        unique=False)
+    op.create_index(
+        op.f('deployments__tenant_id_idx'),
+        'deployments', ['_tenant_id'],
+        unique=False)
+    op.create_index(
+        op.f('events__tenant_id_idx'), 'events', ['_tenant_id'], unique=False)
+    op.create_index(
+        op.f('executions__tenant_id_idx'),
+        'executions', ['_tenant_id'],
+        unique=False)
+    op.create_index(
+        op.f('logs__tenant_id_idx'), 'logs', ['_tenant_id'], unique=False)
+    op.create_index(
+        op.f('nodes__tenant_id_idx'), 'nodes', ['_tenant_id'], unique=False)
+    op.create_index(
+        op.f('node_instances__tenant_id_idx'),
+        'node_instances', ['_tenant_id'],
+        unique=False)
+    op.create_index(
+        op.f('plugins__tenant_id_idx'),
+        'plugins', ['_tenant_id'],
         unique=False)
+    op.create_index(
+        op.f('snapshots__tenant_id_idx'),
+        'snapshots', ['_tenant_id'],
+        unique=False)
+    op.create_index(
+        op.f('secrets__tenant_id_idx'),
+        'secrets', ['_tenant_id'],
+        unique=False)
+
+    # removing duplicated indexes
```

Comment:
checking that every index was indeed duplicated...
i am going to paste the \d for every table as is in 4.5 before this change, and note that the index is created twice